### DISCUSSION TITLE
Does ESPnet work on NVIDIA A100 GPU?

### DISCUSSION THREAD

**Discussion by b1nch3f** [espnet/espnet#4496] [Q&A] [answered]
I'm thinking of spinning a VM in Azure, which has NVIDIA A100 8 GPU, want to know if ESPnet VITS model would train there on LJSpeech dataset.

**Comment by b-flo** [comment, marked as answer]
Hi,

Usually, the incompatibility comes from the dependencies of a task (e.g.: `warp-ctc` or `warp-transducer` for ASR task).
In the case of ViTS, there should be no issue training with an A100! I'll let @kan-bayashi correct me if that's not the case.

**Reply by kan-bayashi** [reply]
As @b-flo said, no problem at all.
I have checked with A6000, which maybe the same as A100.
